panther_driver/src/driver_node.py

```diff
@@ -296,34 +296,21 @@ def driverNode():
                     temp_bat1 = voltage_to_deg(V_temp_bat1)
                     temp_bat2 = voltage_to_deg(V_temp_bat2)
 
-                    V_diff = V_bat1 - V_bat2
-
-                    if abs(V_diff) <= 0.2:
-                        k = 0.5
-                    elif V_diff > 0.2:
-                        k = 1
-                    elif V_diff < -0.2:
-                        k = 0
-                    else:
-                        rospy.logger(f"[{rospy.get_name()}] V_difff out of range")
-
-                    Ibat1 = -1 * ( Idriv1 + (k * Idig) - Icharge_bat1)
-                    Ibat2 = -1 * ( Idriv2 + ((1-k) * Idig) - Icharge_bat2 )
+                    # rospy.loginfo(f"[{rospy.get_name()}] BATTERY LOG:\n" +
+                    #     f"I_bat1={I_bat1:.2f}, V_bat1={V_bat1:.2f}, Idriv1={Idriv1:.2f}, V_driv1={V_driv1:.2f}, I_charge_bat1={I_charge_bat1:.2f}, temp_bat1={temp_bat1:.2f} \n" +
+                    #     f"I_bat2={I_bat2:.2f}, V_bat2={V_bat2:.2f}, Idriv2={Idriv2:.2f}, V_driv2={V_driv2:.2f}, I_charge_bat2={I_charge_bat2:.2f}, temp_bat2={temp_bat2:.2f}")
```

Comment:
Why are we leaving commented out parts of code?

Reply:
That was left out for testing purposes as its likely to be used at this stage. I thouht about cleanning this up during refactor as this data should be looged somehow. What do you think about that?

Reply:
OK